avoid using escape sequences in echo as they're not portable

https://bugs.gentoo.org/523874

--- a/src/man.c
+++ b/src/man.c
@@ -626,7 +626,7 @@ make_roff_command (const char *path, const char *file) {
 		 the current position plus 10 lines.  This plus setpl()
 		 gives us a single page that just contains the whole
 		 man page. (William Webber, wew@cs.rmit.edu.au) */
-	      strcat(buft, "; echo \".\\\\\\\"\"; echo \".pl \\n(nlu+10\"");
+	      strcat(buft, "; echo \".\\\\\\\"\"; echo \".pl \"; echo \"(nlu+10\"");
 #if 0
 	      /* In case this doesnt work for some reason,
 		 michaelkjohnson suggests: I've got a simple
